### ISSUE TITLE
Consider using the echo area for arglists?

### ISSUE THREAD

**Issue by sjl** [l04m33/vlime#25]
I love having the arglist buffer, but it's a little annoying that it's an actual buffer/window that I have to manage (e.g. when I `<c-w>H` to move a window to the left side of the screen it also resizes the arglist window).  Would it be possible to do something like [echodoc](https://github.com/Shougo/echodoc.vim) does to put the arglist in the echo area instead?  If not, no worries.

**Comment by l04m33** [comment]
This is actually a good idea. I've been thinking about this, but didn't figure out how to move the `-- INSERT --` text and other excessive messages out of the way. Requiring the user to manually set `noshowmode` or `cmdheight=2` like `echodoc` does may be a solution though. I'll see what I can do.

**Comment by sjl** [comment]
Yeah I think it's reasonable to require the user to change those settings, rather than trying to do crazy workarounds.

**Comment by jlucas** [comment]
I agree that the three-line tax on the top of the screen isn't ideal.  Check out how [slimv](https://github.com/kovisoft/slimv) is doing it.  You get your "-- INSERT --" mode notifiers, but as soon as you press space after typing a function name you get the arglist.  When you eval the form, or change modes you get your mode notifier back.  It feels natural to me, anyway.  Thanks for vlime.
![slimv](https://user-images.githubusercontent.com/1001845/33237239-0819f0dc-d23c-11e7-94d5-2971fe01a651.jpg)
